DBSetup/data_insertion.sql

```diff
@@ -1,12 +1,20 @@
-
 \c va_trial
 
+insert into Patient values
+(00001, FALSE,'Bar', 'Movshovich', 'movshov@pdx.edu', 'Were no strangers to love You know the rules and so do I A full commitment what Im thinking of You wouldnt get this from any other guy'), 
+(00002, TRUE, 'Elijah', 'Rich-Wimmer', 'elij2@pdx.edu', 'Sweet dreams are made of this Who am I to disagree? I travel the world And the seven seas, Everybodys looking for something.'),
+(00003, FALSE, 'Dominic', 'Lastname', 'fakeemail@pdx.edu', 'Its early morning, the sun comes out Last night was shaking and pretty loud My cat is purring, it scratches my skin So what is wrong with another sin?'),
+(00004, TRUE, 'Liam', 'Wynn', 'wynnliam@pdx.edu', 'Sometimes I feel Ive got to Run away Ive got to Get away from the pain that you drive into the heart of me The love we share Seems to go nowhere And Ive lost my light For I toss and turn I cant sleep at night'),
+(00005, FALSE, 'Griffin', 'Dor', 'harrypotter@gmail.com', 'Woah, were half way there Woah, livin on a prayer Take my hand, well make it I swear Woah, livin on a prayer'),
+(00006, FALSE, 'Mishal', 'Alajmi', 'mialajmi@pdx.edu', 'Layla, youve got me on my knees. Layla, Im begging, darling please. Layla, darling wont you ease my worried mind.'),
+(00007, TRUE, 'Dante', 'inferno', 'devilmaycry@gmail.com', 'Oh, Momma, Im in fear for my life from the long arm of the law. Lawman has put an end to my runnin, and Im so far from my home. Oh, Momma, I can hear you a cryin; youre so scared and all alone. Hangman is comin down from the gallows, and I dont have very long.');
+
 insert into SessionKeys values
 ('0', 00001, 1556100000, 1556200000),
 ('1', 00002, 1556000000, 1557000000),
 ('2', 00003, 1558000000, 1559000000),
 ('3', 00004, 1559000000, 1560000000),
-('4f', 00005, 1560000000, 1561000000),
+('4', 00005, 1560000000, 1561000000),
 ('cafebabe', 00006, 1562000000, 1563000000),
 ('1122334455667788', 00007, 1564000000, 1565000000);
```

Comment:
I like the idea of session keys, but will this be a security issue if we put viewable session keys up to remote?
Could someone just look at this file and get a "good" session key for our DB?

Reply:
These were just for getting a foothold on setting up the session key table. Do we know if Griffin finished the creation of automating session keys? If he did then i can absolutely leave this table empty.